conf/modules.config

```diff
@@ -14,28 +14,28 @@ process {
     publishDir = [
         path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
         mode: params.publish_dir_mode,
-        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        saveAs: { filename -> filename.equals('software_versions.yml') ? null : filename }
```

Comment:
Ah, you're actually checking if it's a yml file. Then please adjust the docu as well and the testdata